snap/README.md

```diff
@@ -532,61 +556,118 @@ root@edgex:~# cd edgex-go && snapcraft --destructive-mode
+After building the snap from one of the above methods, you will have a binary snap package called `edgexfoundry_<latest version>_<arch>.snap`, 
+which can be installed locally with the `--devmode` flag:
 
-```bash
+```
 $ sudo snap install --devmode edgexfoundry*.snap
 ```
 
-In addition, if you are using snapcraft with multipass VM's, you can speedup development by not creating a *.snap file and instead running in "try" mode . This is done by running `snapcraft try` which results in a `prime` folder placed in the root project directory that can then be "installed" using `snap try`. For example:
+In addition, if you are using snapcraft with multipass VM's, you can speedup development by using `snapcraft try` and `snap try`:
+
+1. Clean your multipass VM
 
-```bash
-$ snapcraft try # produces prime dir instead of *.snap file
-...
-You can now run `snap try /home/ubuntu/go/src/github.com/edgexfoundry/edgex-go/prime`.
-$ sudo snap try --devmode prime # snap try works the same as snap install, but expects a directory
-edgexfoundry 1.0.0-20190513+0620a8d1 mounted from /home/ubuntu/go/src/github.com/edgexfoundry/edgex-go/prime
-$
+```
+$ snapcraft clean
+```
+
+2. Create a snap file with a prime folder placed in the root project directory
+
+```
+$ snapcraft prime --shell
+```
+
+It produces prime directory instead of *.snapfile, and copies its prime directory to the current working directory on the host system - 
+outside of any build environment container.
+
+3. Install an unpacked snap using a bind mount to test out the snap defined in the project directory
+
+```
+$ sudo snap try prime --devmode
+```
+
+`snap try` works the same as `snap install`, but expects a directory. 
+
+4. Change your code, then repeat step 2 and 3
+
+5. Ship the tested snap file
+
+```
+$ sudo snap pack ./prime
 ```
 
 #### Interfaces
 
-After installing the snap, you will need to connect interfaces and restart the snap. The snap needs the `hardware-observe`, `mount-observe`, and `system-observe` interfaces connected. These are automatically connected using snap store assertions when installing from the store, but when developing the snap and installing a revision locally, use the following commands to connect the interfaces:
+After installing the snap, you will need to connect interfaces and restart the snap. 
+Interfaces are automatically connected using snap store assertions when installing from the store, 
+but when developing the snap and installing a revision locally, use the commands in this section to connect the interfaces.
+
+To see which interfaces the snap is using, and which interfaces it could use but isn’t:
 
-```bash
-$ sudo snap connect edgexfoundry:hardware-observe
-$ sudo snap connect edgexfoundry:system-observe
-$ sudo snap connect edgexfoundry:mount-observe
```

Comment:
Are `hardware-observe`, `mount-observe`, and `system-observe` no longer needed?

Reply:
No, because [snapcraft.yaml ](https://github.com/edgexfoundry/edgex-go/blob/main/snap/snapcraft.yaml#L68) idn't defined [these 3 plugs and slots](https://snapcraft.io/docs/supported-interfaces).